pages/index.html

```diff
@@ -17,112 +17,87 @@
-    ways. We are also looking for sponsors who want to contribute financially
-    through the <a href="/consortium">Coq Consortium</a>. If you are interested,
-    please get in touch!</p>
-</div>
 
-<div class="frameworklinks">
-<ul>
-<li><a href="/bugs/">Bug tracker</a></li>
-<li><a href="https://github.com/coq/coq/blob/master/CONTRIBUTING.md">Contributing guide</a></li>
-<li><a href="/consortium">Coq Consortium</a></li>
-</ul>
-</div>
-</div>
+<p>Coq has a strong and active user community. They are contributing
+formal developments, extensions of Coq
+(see <a href="/packages.html">Coq Package Index</a>), and tools based
+on Coq (see <a href="/related-tools.html">Related Tools</a>).  We have
+a (multi-lingual) <a href="https://coq.discourse.group">Discourse
+forum</a> and a <a href="https://coq.zulipchat.com">Zulip chat</a>.
+</p>
 
-<div class="framework">
-<div class="frameworklabel"><a  style="color:black;font-weight:bold" href="/documentation">Documentation</a></div>
-<div class="frameworkcontent">
-  <p>The reference documentation for Coq are the <a
-  href="/distrib/current/refman/"> Reference Manual </a> and the
-  documentation of the <a href="/distrib/current/stdlib/">
-  Standard Library</a>. Other useful documents
-  (tutorials, faq, ...) are available from the <a
-  href="/documentation">documentation</a>
-  page.</p>
 </div>
 
 <div class="frameworklinks">
 <ul>
-<li><a href="/distrib/current/refman/"> Reference Manual </a></li>
-<li><a href="/distrib/current/stdlib/"> Standard Library </a></li>
-<li><a href="/documentation">All Documents</a></li>
+<li><a href="/packages.html">Coq Package Index</a></li>
+<li><a href="https://coq.discourse.group">Discourse forum</a></li>
+<li><a href="https://coq.zulipchat.com">Zulip chat</a></li>
+<li><a href="/community.html">Community</a></li>
 </ul>
 </div>
-</div>
-
+</div><!-- /framework -->
 
 <div class="framework">
-<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/community.html">The Coq Community</a></div>
+<div class="frameworklabel"><a style="color:black;font-weight:bold" href="https://github.com/coq/coq/blob/master/CONTRIBUTING.md">Contribute to Coq</a></div>
 <div class="frameworkcontent">
-
-<p>There is a strong and active community of users working with
-Coq. They are contributing formal developments,
-extensions of Coq
-(see <a href="/packages.html">Coq Package Index</a>),
-and tools based on Coq
-(see <a href="/related-tools.html">Related Tools</a>).
-We have a (multi-lingual) <a href="https://coq.discourse.group">Discourse forum</a>
-and a <a href="https://coq.zulipchat.com">Zulip chat</a>.
-</p>
-
+    <p>You can contribute to the development of Coq by reporting bugs,
+    submitting pull requests, improving the documentation, and in many other
+    ways. We are also looking for sponsors who want to contribute financially
+    through the <a href="/consortium">Coq Consortium</a>. If you are interested,
+    please get in touch!</p>
```

Comment:
I'm not sure what the current status is. There might have been a US foundation involved in the process at some point (since most initial contributors were from the US) but I'm not sure it's still the case today.

Reply:
For consortium members such as members of the DeepSpec project, their payments would be considered business expenses--fully deductible without a foundation.  The non profit would benefit individuals making charitable contributions who aren't getting a personal benefit it return.